src/App.js

```diff
@@ -1,13 +1,30 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
```
import React from "react";
```

Remember to remove your unused imports after the code that uses them moves to different components :)